src/Compilers/CSharp/Portable/Binder/Binder_Lookup.cs

```diff
@@ -1449,6 +1463,29 @@ private void MergeHidingLookupResults(LookupResult resultHiding, LookupResult re
             {
                 resultHiding.MergePrioritized(resultHidden);
             }
+
+            return;
+
+            static bool isInExtensionHierarchy(NamedTypeSymbol hiddenContainer, NamedTypeSymbol hidingContainer, ConsList<TypeSymbol> basesBeingResolved, CSharpCompilation compilation, ref CompoundUseSiteInfo<AssemblySymbol> useSiteInfo)
```

Comment:
>isInExtensionHierarchy [](http://example.com/codeflow?start=24&length=22)

Could you please add a comment explaining expected behavior of the method, i.e. stating the intent?

Reply:
Perhaps the intent would be obvious if the helper was named something like "isOrDerivedFromExtendedType".